packages/fxa-auth-server/lib/push.js

```diff
@@ -473,6 +473,7 @@ module.exports = function (log, db, config, statsd) {
       } else {
         errCode = ERR_PUSH_UNKNOWN;
       }
+      err.errCode = errCode;
```

Comment:
This is to help ensure that we return one of a fixed set of strings to the client, rather than an arbitrary `Error` instance.